driver/xrt/CMakeLists.txt

```diff
@@ -72,7 +72,7 @@ target_link_libraries(accl PUBLIC zmqpp)
 find_package(jsoncpp REQUIRED)
 target_link_libraries(accl PUBLIC jsoncpp_lib)
 get_target_property(JSON_INC_PATH jsoncpp_lib INTERFACE_INCLUDE_DIRECTORIES)
-target_include_directories(accl PUBLIC ${JSON_INC_PATH})
```

Comment:
Are you sure this is correct? For me `JSON_INC_PATH` contains the path `/usr/include/jsoncpp`, while `JSONCPP_INCLUDE_DIRS` is empty. `jsoncppConfig.cmake` specifically sets the line
```
set_target_properties(jsoncpp_lib PROPERTIES
  INTERFACE_INCLUDE_DIRECTORIES "/usr/include/jsoncpp"
)
```
which we then retrieve and store in `JSON_INC_PATH`.

Reply:
I was using FindJsonCpp, which is not part of the default modules of CMake... so you are right, I will revert this.